docs/modules/user-guide/partials/proc_creating-a-devfile-stack.adoc

```diff
@@ -0,0 +1,18 @@
+[id="creating-a-devfile-stack_{context}"]
+= Creating a Devfile stack
+
+[role="_abstract"]
+Create a devfile stack to package into the devfile registry.
+
+.Procedure
+
+. Create a stack folder with a name matches the stack name, for example, `java-wildfly/`.
```

Comment:
`Create a stack folder with a name that matches the stack name. For example, `java-wildfly/`.`

Reply:
I think it reads better with the `that` included and a separate sentence for the example :)